doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -684,6 +684,10 @@ Libraries for Zigbee
       *  ``zdo`` shell commands extended to allow binding to a group address.
       * Internal context manager structures.
 
+    * Fixed:
+
+      * The issue where Zigbee generic zcl cmd uses incorrect index of a context manager entry during cleanup after the command was sent.
```

Comment:
```suggestion
      * The issue where the generic Zigbee `zcl cmd` was using the incorrect index of a context manager entry during cleanup after the command was sent.
```

Reply:
I've changed the suggestion a little bit so it is similar to what we have in `Updated` section:
`The issue where the ``zcl cmd`` shell command was using the incorrect index of a context manager entry during cleanup after the command was sent.`